azure-stack/operator/azure-stack-download-azure-marketplace-item.md

```diff
@@ -82,26 +82,28 @@ There are two parts to this scenario:
 
   - To enable import of a downloaded marketplace item, the [PowerShell environment for the Azure Stack Hub operator](azure-stack-powershell-configure-admin.md) must be configured.
 
-  - Clone the [Azure Stack Hub tools](https://github.com/Azure/AzureStack-Tools) GitHub repo.
+- Download the Azs.Syndication.Admin Module from the PowerShell Gallery using the command below
+  ```
+  Install-Module -Name Azs.Syndication.Admin
+  ```
 
-- You must have a [storage account](azure-stack-manage-storage-accounts.md) in Azure Stack Hub that has a publicly accessible container (which is a storage blob). You use the container as temporary storage for the marketplace items gallery files. If you're not familiar with storage accounts and containers, see [Work with blobs - Azure portal](/azure/storage/blobs/storage-quickstart-blobs-portal) in the Azure documentation.
-
-- The marketplace syndication tool is downloaded during the first procedure.
-
-- You can install [AzCopy](/azure/storage/common/storage-use-azcopy) for optimal download performance, but it's not required.
-
-Once you have registered, you can disregard the following message that appears on the Marketplace management blade, as this is not relevant for the disconnected use case:
+Once you have registered your Azure Stack, you can disregard the following message that appears on the Marketplace management blade, as this is not relevant for the disconnected use case:
 
 ![Marketplace management](media/azure-stack-download-azure-marketplace-item/toolsmsg.png)
 
 ### Use the marketplace syndication tool to download marketplace items
 
 > [!IMPORTANT]
-> Be sure to download the marketplace syndication tool each time you download marketplace items in a disconnected scenario. Frequent changes are made to this script and the most current version should be used for each download.
+> Be sure to download the marketplace syndication tool each time you download marketplace items in a disconnected scenario. Frequent changes are made to this tool and the most current version should be used for each download.
 
 1. On a computer with an Internet connection, open a PowerShell console as an administrator.
 
-2. Add the Azure account that you've used to register Azure Stack Hub. To add the account, in PowerShell run **Add-AzureRmAccount** without any parameters. You are prompted to enter your Azure account credentials and you might have to use two-factor authentication, depending on your account configuration.
+2. Sign in to the appropriate Azure cloud and AAD Directory Tenant using the Azure account that you've used to register Azure Stack Hub. To add the account, in PowerShell run **Add-AzureRmAccount**. 
+
```

Comment:
We can't use the AAD acronym, it has to be "Azure AD"

Reply:
Done.